ci.nix

```diff
@@ -19,17 +19,21 @@ in
 dimension "System" (systems genericPkgs) (systemName: system:
   let
     packageSet = import ./default.nix { inherit system rev; checkMaterialization = true; };
-    packages = import ./nix { inherit system rev; checkMaterialization = true; };
-    pkgs = packages.pkgs;
-    pkgsLocal = packages.pkgsLocal;
+    pkgs = packageSet.pkgs;
+    pkgsLocal = packageSet.pkgsLocal;
+    pkgsMusl = packageSet.pkgsMusl;
     lib = pkgs.lib;
     platformFilter = platformFilterGeneric pkgs system;
   in
   filterAttrsOnlyRecursive (_: v: platformFilter v) {
     inherit (packageSet) docs papers tests plutus-playground marlowe-playground plutus-scb marlowe-symbolic-lambda;
     inherit (pkgsLocal.haskell.project) roots;
+
     # build the shell expression to be sure it works on all platforms
-    shell = import ./shell.nix { };
+    shell = import ./shell.nix {
+      packages = { inherit pkgs pkgsMusl pkgsLocal; };
```

Comment:
`packages = packageSet`? or rename `packageSet` to `packages` and `inherit packages`?

Reply:
Yeah let's just rename `packageSet` to `packages` :+1: